### PR TITLE
improve ads plugin

### PR DESCRIPTION
**Pull request by alexzheng**
update admob SDK and APIs
interstitial support preload and refresh when closed
fix setAlignment issue.

### CODE REVIEW

File: plugins/ads/source/Android/src/com/giderosmobile/android/plugins/ads/AdsStateChangeListener.java [hunk 1]
```diff
@@ -4,4 +4,5 @@
 	public void onShow();
 	public void onDestroy();
 	public void onHide();
+	public void onRefresh();
```
Comment (hgy29) [inline plugins/ads/source/Android/src/com/giderosmobile/android/plugins/ads/AdsStateChangeListener.java]: This new onRefresh method must be implemented by all ads provider. Merging this change right now will break all non modified ads providers. Same remark goes for android

Comment (alexzheng) [inline plugins/ads/source/Android/src/com/giderosmobile/android/plugins/ads/AdsStateChangeListener.java]: oh yes, I use admob only, I have not compiled any others.

**Comment by alexzheng** [comment]
added for all

**Comment by hgy29** [comment]
Great thanks! Shouldn't the same be done for iOS ?

**Comment by alexzheng** [comment]
I think it's not needed for iOS

**Comment by alexzheng** [comment]
OK, removed

**Comment by hgy29** [comment]
Hi @alexzheng, a few people are complaining regrading ads changes on the forum: http://giderosmobile.com/forum/discussion/comment/50504#Comment_50504

Can you answer them ?

**Pull request merged**